ia/src/game.py

```
@@ -32,11 +32,11 @@ def cleanup_thread() -> None :
 def game(process_args) -> None :
   global threads
 
-  server_adress = (process_args.machine, process_args.port)
+  server_address = (process_args.machine, process_args.port)
```

Pylint (3.3.7)

[warning] Bad indentation. Found 2 spaces, expected 4

(W0311)